src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -11,24 +11,37 @@
 import seedu.address.model.Model;
 import seedu.address.model.pdf.Pdf;
 
+import java.io.File;
+import java.nio.file.Paths;
+
 /**
  * Deletes a pdf identified using it's displayed index from the pdf book.
  */
 public class DeleteCommand extends Command {
 
     public static final String COMMAND_WORD = "delete";
 
+    public enum DELETE_TYPE {
+        HARD, SOFT
+    }
+
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the PDF identified by the index number used in the displayed PDF list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "Hard or soft delete can be chosen (soft delete selected by default).\n"
```

Comment:
I think we can remove the soft delete can be chosen part? Cause its by default.

Reply:
Also I think must specify what hard delete does? Or do we just do it in the UG & DG?